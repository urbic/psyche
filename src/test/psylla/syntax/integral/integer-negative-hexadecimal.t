-0x2026 ?
-0X2026 ?
