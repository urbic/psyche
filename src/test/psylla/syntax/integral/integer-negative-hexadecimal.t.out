-8230
-8230
